docs/_includes/chips-combobox.html

```diff
@@ -0,0 +1,396 @@
+    <p>This is the chips combobox with pre-populated chip items, something you'd see when editing a chips combobox component.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="chips-combobox">
+                <div class="chips-combobox__items" role="group" aria-describedby="chips-combobox-1-heading">
+                    <span class="chip">
+                        <span id="chip-interactive-1-1-text" class="chip__text">Football</span>
+                        <button class="chip__button" type="button" aria-label="Remove" aria-describedby="chip-interactive-1-1-text">
+                            <svg class="icon icon--close-12" focusable="false" width="13" height="12" aria-hidden="true">
+                                {% include symbol.html name="close-12" %}
+                            </svg>
+                        </button>
+                    </span>
+                    <span class="chip">
+                        <span id="chip-interactive-1-2-text" class="chip__text">Basketball</span>
+                        <button class="chip__button" type="button" aria-label="Remove" aria-describedby="chip-interactive-1-2-text">
+                            <svg class="icon icon--close-12" focusable="false" width="13" height="12" aria-hidden="true">
+                                {% include symbol.html name="close-12" %}
+                            </svg>
+                        </button>
+                    </span>
+                    <span class="chip">
+                        <span id="chip-interactive-1-3-text" class="chip__text">Baseball</span>
+                        <button class="chip__button" type="button" aria-label="Remove" aria-describedby="chip-interactive-1-3-text">
+                            <svg class="icon icon--close-12" focusable="false" width="13" height="12" aria-hidden="true">
+                                {% include symbol.html name="close-12" %}
+                            </svg>
+                        </button>
+                    </span>
+                </div>
+                <span class="combobox combobox--js chips-combobox__combobox">
+                    <span class="combobox__control chips-combobox_combobox__control">
+                        <input id="chips-combobox-1-input" role="combobox" type="text" placeholder="Add Sport" aria-haspopup="listbox" aria-owns="listbox-chips-combobox-1" />
+                        <svg class="icon icon--chevron-down-16" focusable="false" height="16" width="16" aria-hidden="true">
+                            {% include symbol.html name="chevron-down-16" %}
+                        </svg>
+                    </span>
+                    <div class="combobox__listbox">
+                        <div id="listbox-chips-combobox-1" class="combobox__options" role="listbox">
+                            <div class="combobox__option" role="option">
+                                <span>Baseball</span>
```

Comment:
Being picky here, but Baseball is already selected as a chip (and so shouldn't be visible as a combobox option), so let's replace it with another sport.

<img width="657" alt="Screenshot 2023-12-18 at 2 03 10 PM" src="https://github.com/eBay/skin/assets/38065/a5e74b54-54c8-4472-be93-28ee7c15c195">